.github/workflows/UpdatePropertiesTableInMSDocs.yaml

```diff
@@ -0,0 +1,99 @@
+# This workflow will create PR to update the configuration properties files in MS docs repository: https://github.com/MicrosoftDocs/azure-dev-docs-pr/tree/main/articles/java/spring-framework/includes/spring-cloud-azure
+# For example, this PR is created by this workflow: https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3035
+
+name: Update the configuration properties files in MS docs repo
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  update-change:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build latest azure-sdk-for-java repository
+        run: |
+          git init azure-sdk-for-java
+          cd azure-sdk-for-java
+          git remote add origin https://github.com/Azure/azure-sdk-for-java.git
+          git config core.sparsecheckout true
+          echo "sdk/spring" >> .git/info/sparse-checkout
+          echo "eng" >> .git/info/sparse-checkout
+          echo "sdk/keyvault" >> .git/info/sparse-checkout
+          echo "sdk/boms" >> .git/info/sparse-checkout
+          echo "sdk/aot" >> .git/info/sparse-checkout
+          git pull --depth=1 origin main
+          mvn clean install -Dmaven.javadoc.skip=true -DskipTests \
+            -Dcheckstyle.skip=true \
+            -ntp \
+            -Dspotbugs.skip=true \
+            -Drevapi.skip=true -Djacoco.skip=true \
+            -Dparallel-test-playback \
+            -Pdev \
+            -f sdk/spring/pom.xml
+
+      - uses: actions/checkout@v3
+        name: Checkout microsoft/spring-cloud-azure
+
+      - name: Generate configuration properties files
+        run: |
+          mkdir spring-cloud-azure-markdown-appdendix-generator/src/main/asciidoc
```

Comment:
This is not necessary.

Reply:
When the directory `asciidoc` cannot be found, the file will not be created